app/views/homes/show.html.erb

```diff
@@ -15,52 +15,68 @@
-                                    <input class="search-button button no-shadow postfix" data-disable-with="Searching..." type="submit" value="<%= t('home.search') %>">
-                                </div>
-                            </div>
+                <div class="search-bar-container">
+                    <div class="search-bar" ng-app="searchApp" ng-controller="searchCtrl">
+                        <div class="search-input-container">
+                            <input typeahead-min-length="1" autocomplete="off" class="search-input ng-pristine ng-valid ng-valid-maxlength ng-touched" id="q" maxlength="120" name="q" ng-change="search()" ng-model="query" placeholder="<%= t('home.placeholder') %>" type="text" typeahead-wait-ms="200" typeahead="crop.name for crop in crops">
                         </div>
-                    </div>
-                    <div class="small-12">
-                        <div class="add-a-guide text-center">
-                            <p>
-                                Share your knowledge with the world, <%= link_to "create a guide", new_guide_path, class:"create-guide-link" %>.
-                            </p>
+                        <div class="search-button-container">
+                            <input class="search-button button no-shadow postfix" data-disable-with="Searching..." type="submit" value="<%= t('home.search') %>">
                         </div>
                     </div>
                 </div>
             </form>
         </div>
-        <a href="#explore-community-favorites" id="explore-community-favorites" class="explore-community-favorites text-center">
-            <h2><%= t('home.explore_community_favorites') %></h2>
-            <i class="fa fa-angle-down"></i>
-        </a>
+    </div>
+    <div class="how-it-works" id="how-it-works">
+        <div class="row">
+            <a href="#"><i class="fa fa-times exit"></i></a>
+            <div class="small-12 columns steps">
+                <div class="small-12 medium-4 columns">
+                    <div class="step step-circle step-circle-1">1</div>
+                        <%= t 'home.step_1' %>
+                    <div class="step-icon step-1"></div>
+                </div>
+                <div class="small-12 medium-4 columns">
+                    <div class="step step-circle step-circle-2">2</div>
+                        <%= t 'home.step_2' %>
+                    <div class="step-icon step-2"></div>
+                </div>
+                <div class="small-12 medium-4 columns">
+                    <div class="step step-circle step-circle-3">3</div>
+                        <%= t 'home.step_3' %>
+                    <div class="step-icon step-3"></div>
+                </div>
+            </div>
+            <h2 class="center guide-description">
+                <%= t 'home.guide_description' %>
+            </h2>
+            <ul class="life-stage-icons">
+                <!-- Why don't these icons show up? -->
```

Comment:
Woops, thats from an outdated diff. What I couldn't get to work originally was something like: `<img src="stages/prep.png">` Never figured it out so I just did the CSS background image thang instead